glean-core/src/database/mod.rs

```diff
@@ -81,7 +110,12 @@ impl Database {
         let path = Path::new(data_path).join("db");
         log::debug!("Database path: {:?}", path.display());
 
+        // FIXME: This is probably more knowledge than we should have about the database internals.
+        // We could instead iterate over the directory and sum up the file sizes.
```

Comment:
This seems like a reasonable fix. Would you kindly file a bug and add the number here?

Reply:
yup, will do